hare/eligibility/oracle.go

```diff
@@ -408,11 +396,6 @@ func (o *Oracle) actives(ctx context.Context, targetLayer types.LayerID) (map[st
 		log.FieldNamed("target_layer_epoch", targetLayer.GetEpoch()))
 	logger.Debug("hare oracle getting active set")
 
-	// we can't read blocks during genesis epochs as there are none
-	if targetLayer.GetEpoch().IsGenesis() {
```

Comment:
consider adding a similar conditional on [this line](https://github.com/spacemeshos/go-spacemesh/pull/2518/files#diff-a2fa964ab3766e7996ed48abf1f4db1a8f864e897bf510e899fc5dd3d4d131d0R473) so that a warning is not printed during genesis (maybe just INFO)

Reply:
Your link is not to anything specific so I don't really understand what you're referring to... where would a warning be printed in genesis? IIUC nothing in the code should call `actives()` in genesis anyway, so this code was zombie code before this change. Maybe I missed something - I can look at the logs to check.

Reply:
I made the change myself, take a look: cbf0623